### DISCUSSION TITLE
Create Payment Order necessite d'être capturé

### DISCUSSION THREAD

**Discussion by sullivan-iteracode** [PayGreen/documentation#17] [Q&A] [answered]
Bonjour,

J'ai créé un Payment Order avec :
- Platforms: conecs
- AutoCapture: true

Cependant pour les paiements avec un ticket resto le paiement n'est pas capturé automatiquement. L'utilisateur doit cliquer sur le bouton capture dans l'interface du paiement.

**Comment by enzobillisPG** [comment, marked as answer]
Bonjour @sullivan-iteracode 
Les paiements via la plateforme Conecs ont un délais de capture automatique de 48H. En effet les remboursements Conecs sont possibles uniquement **avant** la capture. C'est pour cette raison que la capture automatiquement n'est pas instantanée dans ce cas. 
Enzo

**Reply by sullivan-iteracode** [reply]
d'accord, merci
